tests/run-make/rustdoc-map-file/rmake.rs

```diff
@@ -8,6 +10,48 @@ fn main() {
         .arg("--generate-redirect-map")
         .out_dir(&out_dir)
         .run();
-    // FIXME (GuillaumeGomez): Port the python script to Rust as well.
-    python_command().arg("validate_json.py").arg(&out_dir).run();
+
+    let mut found_file = false;
+    read_dir_entries_recursive(&out_dir, |path| {
+        if !found_file
+            && path.is_file()
+            && path.file_name().map(|name| name == "redirect-map.json").unwrap_or(false)
```

Comment:
Question: are there multiple `redirect-map.json`s generated? Or is the directory structure in which they are going to be generated as unstable / subject to frequent changes?

Reply:
I was simply lazy to look at the right place. ^^'